zospy/zpcore.py

```diff
@@ -282,13 +282,21 @@ class ZOS:
     ----------
         ZOSAPI (None | netModuleObject): The ZOSAPI interface or if loaded.
         ZOSAPI_NetHelper (None | netModuleObject): The ZOSAPI_NetHelper interface if loaded.
+
+    Parameters
+    ----------
+    preload: bool
```

Comment:
This one applies to all docstrings: NumPyDoc uses a space before the colon.
```suggestion
    preload : bool
```